### PR TITLE
Bugfix: Correct Luna’s blowjob background after bedsex

### PR DESCRIPTION
**Pull request by zennad**
Once you have performed the bed sex favour if you select ‘let her do
her own thing’ under the BJ options you will find that the background
image is still Luna’s bedroom.

This seems to be because the CG path is not set to the under-desk folder

### CODE REVIEW

File: game/_script_/_luna_/personal_favours/ll_pf_T3b_blowjob.rpy [hunk 1]
```diff
@@ -681,6 +681,9 @@ label ll_pf_blowjob_T1_slapping: # Call label
 
     ">You quickly take out your hard cock and slap it against Luna's naive nose a few times."
 
+    $ ccg_folder    = "luna_desk"
+    $ ccg("base","blank","blank")
```
Comment (zennad) [inline game/_script_/_luna_/personal_favours/ll_pf_T3b_blowjob.rpy]: I think that these two lines could be removed

**Comment by zennad** [comment]
I have also made a commit on my repo which removes the two lines:

```ren'py
    $ ccg_folder    = "luna_desk"
    $ ccg("base","blank","blank")
```

But it's on a spin-off branch where I'm working on refactoring the CG code so that it doesn't assign all of the CG folder variables at the start of each choice.

My thought is that these can be assigned at the top of the favour and modified as needed through the script.

Anyway, this is just a silly suggestion.

**Comment by zennad** [comment]
Note: There is still a blue haze at the bottom of the screen.

**Comment by TropeCode** [comment]
Thanks for attempting to fix this issue. It seems that the CG code doesn't (re)set all the layers properly which causes old images to persist after certain events. We will likely refactor Luna's CG into a common CG system since we're about to add new CG in the near future anyway.

**Pull request merged**